BankManagementSystem/Controller/CustomerController.cs

```diff
@@ -11,12 +12,70 @@ public void CreateAccount()
 
 
         }
-        public void EditDetails()
+        public void EditDetails(long accountNumber)
         {
+            CustomerModel customerModel;
+            customerModel = (CustomerModel)Program.CustomerTable[accountNumber];
+                Console.WriteLine("Saved account details:");
+                Console.ForegroundColor = ConsoleColor.Blue;
+                ViewDetails(accountNumber);
+                Console.ForegroundColor=ConsoleColor.Green;
+                Console.WriteLine("\nSelect the field you want to update: \n1.Name\n2.DOB\n3.Address\n4.Phone Number\n5.Exit");
+                Console.ForegroundColor = ConsoleColor.White;
+                int choice = int.Parse(Console.ReadLine());
+                switch (choice)
+                {
+                    case 1:
+                        Console.WriteLine("Enter new name:");
+                        string replaceName=Console.ReadLine();
+                        customerModel.CustomerName = replaceName;
+                        break;
+                    case 2:
+                        Console.WriteLine("Enter updated date of birth:");
+                        string updateDOB = Console.ReadLine();
+                        customerModel.CustomerName = updateDOB;
```

Comment:
change the customerModel.customerName to customerModel.CustomerDateOfBirth

Reply:
will check that